python/tvm/relay/frontend/common.py

```diff
@@ -129,6 +129,28 @@ def get_int_tuple(self, key, default=RequiredAttr()):
             raise AttributeError("Required attribute {} not found.".format(key))
         return default
 
+    def get_int_tuple_default(self, key, default_attribute=RequiredAttr(), default_value=None):
```

Comment:
instead of introducing get_int_tuple_default, seems we can just keep get_int_tuple by adding more parameters?